### DISCUSSION TITLE
Activity handbook 🎮

### DISCUSSION THREAD

**Discussion by Dedekind561** [CodeYourFuture/curriculum#162] [Ideas]
I've been thinking a lot about the Saturday sessions and how to make explain what kind of activities we can be doing and why they are important. I've started documenting activities that I think are useful based off what I've observed at CYF, based off existing CYF docs, based on general teaching, some stuff from Teaching Tech Together. It isn't re-inventing the wheel: more just precisely labelling stuff we do all the time without even really thinking about it - but it may well be less obvious to others!  I think having activities documented like this will be very useful for those who want to build more resources like workshops etc.
We need to think about how we expose this in the curriculum UI too.
There are some different categories:

**Nature of the activity**
This is just what the actual activity is:
Are we defining something, finding a bug etc.

**Type of interaction**
The type of interaction defines how we're engaging with others in the community. Are we working as a group? Are we working in pairs etc. 

Here's what I've started: https://docs.google.com/document/d/1QjqQCxS2IFBj-SlKkpLbSa59uJoEdatnyiB00X_glQk/edit

**Comment by SallyMcGrath** [comment]
We should put this as a data source for the curriculum -  I think this should be a ticket

**Comment by alismith-dev** [comment]
I don't have access to the Google Doc, have requested access.

**Reply by SallyMcGrath** [reply]
I've made the document public and given you editor rights, Ali.

**Comment by alismith-dev** [comment]
Following up _super_ late here as I've had the tab open for ages.

This looks like a really great resource and something that I think we should look to include in the "How the Curriculum Works" module that @Dedekind561 has started to sketch out.

**Reply by alismith-dev** [reply]
With that lens, I left a few comments and suggestions :)

**Comment by SallyMcGrath** [comment]
@moneyinthesky Here's a longer lived discussion on this

This turned into https://curriculum.codeyourfuture.io/activities/

These activities can be assigned to blocks and workshops in the front matter. It's a [Hugo taxonomy ](https://gohugo.io/content-management/taxonomies/)

Maybe they can also be flagged up in https://curriculum.codeyourfuture.io/how-our-curriculum-works/
